schemas/context/consent.schema.json

```diff
@@ -0,0 +1,63 @@
+{
+	"meta:license": [
+		"Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+		"This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+		"you may not use this file except in compliance with the License. You may obtain a copy",
+		"of the License at https://creativecommons.org/licenses/by/4.0/"
+	],
+	"$id": "https://ns.adobe.com/xdm/context/consent",
+	"$schema": "http://json-schema.org/draft-06/schema#",
+	"title": "XDM Consent",
+	"type": "object",
+	"auditable": true,
+	"meta:extensible": true,
+	"meta:abstract": true,
+	"meta:extends": [
+		"https://ns.adobe.com/xdm/data/record"
+	],
+	"description": "An XDM Consent is a representation of the consent responses given by end users/consumers of a given brand to allow or not allow the Brand/Customers to use their personal data for analytics purposes like Direct Marketing, First Party personalization, cross device profile consolidation, advertsing, sending offers etc..",
+	"definitions": {
+		"consent": {
+			"properties": {
+				"Id": {
```

Comment:
I am not sure if there is any guideline around the field names. I have mostly seen that they have camel case. It would be good to check if that should be followed

Reply:
Corrected !